lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -57,12 +57,13 @@ def capture(amount, authorization, options = {})
 
       def credit(amount, payment, options = {})
         post = {}
-        post[:instruction] = {}
-        post[:instruction][:funds_transfer_type] = options[:funds_transfer_type] || 'FD'
         add_processing_channel(post, options)
         add_invoice(post, amount, options)
         add_payment_method(post, payment, options, :destination)
         add_source(post, options)
+        add_instruction_data(post, options)
+        add_payout_sender_data(post, options)
+        add_payout_destination_data(post, options)
```

Comment:
Holy cow that is a lot of new fields for payouts. Are these fields reserved only for payouts?

Reply:
So by adding these fields it makes the credit transaction a payout? I asking cause I also that the endpoint doesn't change for payouts